generators/app/templates/src/projectName.jsx

```diff
@@ -16,33 +26,29 @@ class <%= projectClassName %> extends React.Component {
   }
 
   render() {
+    const { name, variant, ...customProps } = this.props;
     const classes = classNames(['<%= projectCssClassName %>',
-      this.props.variant,
+      variant,
       this.state.isSelected && 'u-selected',
+      customProps.className,
     ]);
 
 
-    if (!this.props.name) {
+    if (!name) {
       return null;
     }
-    if (!this.props.variant) {
+    if (!variant) {
       return null;
     }
     return (<button
+      {...customProps}
```

Comment:
So if you pass in className as one of your customProps, will that define it here, and then just be overwritten by the next line?